python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -211,11 +211,13 @@ def _update(self):
         if self.resource_demand_scheduler:
             resource_demand_vector = self.resource_demand_vector + \
                 self.load_metrics.get_resource_demand_vector()
-            to_launch = (self.resource_demand_scheduler.get_nodes_to_launch(
+            placement_group_load = self.load_metrics.get_placement_group_load()
```

Comment:
Please rename this to `pending_placement_groups` here and elsewhere for clarity.